processing/src/main/java/io/druid/segment/IndexIO.java

```diff
@@ -813,4 +813,14 @@ public static File makeMetricFile(File dir, String metricName, ByteOrder order)
   {
     return new File(dir, String.format("met_%s_%s.drd", metricName, order));
   }
+
+  public static void main(String[] args) throws Exception
```

Comment:
While you have the README.md open, maybe you could consider the first line under Documentation, wherethe two links are the same.  Should the link for "project webpage" be http://druid.io? 


     On Friday, March 27, 2015 6:18 PM, Fangjin Yang <notifications@github.com> wrote:
   

 In processing/src/main/java/io/druid/segment/IndexIO.java:> @@ -813,4 +813,14 @@ public static File makeMetricFile(File dir, String metricName, ByteOrder order)
>    {
>      return new File(dir, String.format("met_%s_%s.drd", metricName, order));
>    }
> +
> +  public static void main(String[] args) throws Exception
er, yeah, dont look at this...—
Reply to this email directly or view it on GitHub.